pyface/ui/wx/grid/grid.py

```diff
@@ -1310,50 +1294,25 @@ def __initialize_sort_state(self):
     def __initialize_style_settings(self, event=None):
 
         # make sure all the handlers for traits defining styles get called
-        self._on_enable_lines_changed()
-        self._on_read_only_changed()
-        self._on_selection_mode_changed()
-        self._on_column_label_height_changed()
-        self._on_row_label_width_changed()
-        self._on_show_column_headers_changed()
-        self._on_show_row_headers_changed()
-        self._on_default_cell_bg_color_changed()
-        self._on_default_label_bg_color_changed()
-        self._on_default_label_text_color_changed()
-        self._on_selection_bg_color_changed()
-        self._on_selection_text_color_changed()
+        self._on_enable_lines_changed(event=None)
+        self._on_read_only_changed(event=None)
+        self._on_selection_mode_changed(event=None)
+        self._on_column_label_height_changed(event=None)
+        self._on_row_label_width_changed(event=None)
+        self._on_show_column_headers_changed(event=None)
+        self._on_show_row_headers_changed(event=None)
+        self._on_default_cell_bg_color_changed(event=None)
+        self._on_default_label_bg_color_changed(event=None)
+        self._on_default_label_text_color_changed(event=None)
+        self._on_selection_bg_color_changed(event=None)
+        self._on_selection_text_color_changed(event=None)
```

Comment:
let's just update the method signatures to use `event=None` and we won't need to update the callers. Here and in other places in this module.